static/js/message_list.js

```diff
@@ -42,6 +42,11 @@ exports.MessageList.prototype = {
         var bottom_messages = [];
         var interior_messages = [];
 
+        if (opts.messages_are_new && !self.fetch_status.can_append()) {
+            blueslip.info('ignoring new message for list that is behind on fetches');
```

Comment:
Should this be `blueslip.log` or lower?  I.e. do we want this in the JS console?